ui-core/src/stories/TimePicker.stories.tsx

```diff
@@ -70,11 +68,39 @@ export const Default: Story = {
   args: {
     label: 'Time',
   },
+  decorators: [
+    (Story) => (
+      <div style={{ maxWidth: '6.7rem', minHeight: '500px' }}>
```

Comment:
Why not add this div in the story directly ?

Reply:
I think it’s the purpose of the decarator. that’s what’s done in the rest of the stories. I don’t think it’s worth dealing with more props